community-membership.md

```diff
@@ -12,7 +12,7 @@ delegated. They can be reached via e-mail cncf-opentelemetry-governance@lists.cn
 
 | **Role**   | **Responsibilities**                                  | **Requirements**                                             | **Defined by**                                               |
 | ---------- | ----------------------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
-| member     | active contributor in the community.  reviewer of PRs | sponsored by 2 owners. multiple contributions to the project. | OpenTelemetry GitHub org member.                             |
+| member     | active contributor in the community.  reviewer of PRs | sponsored by 2 approvers or maintainers. multiple contributions to the project. | OpenTelemetry GitHub org member.                             |
```

Comment:
Maintainers across many repos is already a sizeable group. Is there some friction why we need to expand it more, to approvers?

Reply:
Oh, race conditioned. Note that it's tidying up what the criteria _already_ said in https://github.com/GregMefford/community/blob/master/community-membership.md#member below, and summarizing it correctly, rather than changing the criteria.